brands/shop/paint.json

```diff
@@ -34,13 +34,13 @@
       "shop": "paint"
     }
   },
-  "shop/paint|Sherwin Williams": {
-    "matchNames": ["sherwin williams paints"],
+  "shop/paint|Sherwin-Williams": {
+    "matchNames": ["sherwin-williams paints"],
```

Comment:
Fuzzy matching will work regardless of this particular change, but I made the change anyways to clarify that it’s the addition of “paints” that we want to match. 🤷‍♂